include/jemalloc/internal/thread_event.h

```diff
@@ -44,7 +44,8 @@ void thread_event_boot();
     C(thread_allocated_next_event_fast)					\
     C(thread_allocated_last_event)					\
     C(thread_allocated_next_event)					\
-    ITERATE_OVER_ALL_EVENTS
+    ITERATE_OVER_ALL_EVENTS						\
+    C(prof_sample_last_event)
```

Comment:
IIRC, the last_event isn't required for other events (only to facilitate idump?). If that's the case, let's not add it to the list and manually invoke the getter / setter (I think call the tsd ones directly is fine).

Reply:
The only thing is that the getter and setter for it would be the same as those for the other counters, but different from those for other `tsd` fields in that the setter needs to invoke the pointer getter so as to work when `tsd_nominal(tsd)` is `false`.

Reply:
Okay let’s keep it as is for now.